src/message/html/js.js

```diff
@@ -63,28 +63,20 @@ window.addEventListener('scroll', () => {
 });
 
 document.body.addEventListener('click', e => {
-  alert('click');
-  sendMessage({
-    type: 'click',
-    target: e.traget,
-    targetNodeName: e.target.nodeName,
-    targetClassName: e.target.className,
-    matchez: e.target.matches('a[target="_blank"] > img'),
-  });
-
   if (e.target.matches('.avatar-img')) {
     sendMessage({
       type: 'avatar',
       fromEmail: e.target.getAttribute('data-email'),
     });
+    return;
   }
 
-  if (e.target.matches('.header')) {
```

Comment:
That is definitely a correct way to handle this. If some of the html rendered does not have the 'header' class, then this is where the change should happen!